Gaijinizer patch/Map061.txt

```diff
@@ -118,18 +129,20 @@ control!"#MTLed
 やつは命令より『ショー』を優先するクソガキだ。
 アテにはできねぇ。
 > CONTEXT: Map061/events/5/pages/0/84/Dialogue/other2
-............ "E-002" is indeed strong,
-He is a slogan who gives priority to "show" over
-orders. I can not do it.#MTLed
+... Sure E-002's strong.
+But she's also a stupid brat who prioritizes her 
+"show" over following orders. Real unreliable.#Ron
 > END STRING
 
 > BEGIN STRING
 仮にそのあと、E-003があの男の始末をつけたとして、だぞ？
 
 いや、間違いなくE-003なら喰っちまうだろうな…。
 > CONTEXT: Map061/events/5/pages/0/89/Dialogue/other2
-If, after that, E-003 got rid of a man, right?
-No, I'm sure E-003 will get lost ....#MTLed
+Let's say after that E-003 manages to sort out our
+little issue with that man...
+Actually, I'm 100% sure he'd be monster food with
+with E-003 around...#Ron
```

Comment:
A tiny line overage here, and a slightly better word choice for the second line (Additionally, there are two "with"s):
>_**Let's say after that E-003 manages to sort out
our little issue with that man...
Never mind that, I'm 100% sure he'd be monster food
with E-003 around...**_